StringHandler.java

```diff
@@ -0,0 +1,192 @@
+import java.util.*;
+
+/*
+StringHandler is a class for taking strings as input and doing operations on it
+like reverse, finding the largest word in the string, and changing the
+case of characters from uppercase to lowercase and vice-versa.
+
+@author Saurabh Aswal
+@version 1.0
+@since 20/01/18
+*/
+
+class StringHandler
+{	
+	/*
+	Method for taking a string as an input.
+	@return String type.
+	*/
+	String inputString()
+	{
+		Scanner sc = new Scanner(System.in);
+		String input = sc.nextLine();
+		return input;
+	}
+	
+	/*
+	Method for reversing the input string.
+	@param String type.
+	@return String type.
+	*/
+	String reverse(String s)
+	{
+		int sSize = s.length();
+		
+		String temp = new String();
+		
+		for(int i=sSize-1; i >= 0; i--)
+		{
+			temp = temp + s.charAt(i);
+		}
+		return temp;
+	}
+	
+	int compare(String s1, String s2)
+	{
+		int sSize = s1.length();
+		
+		if(s1.length() == s2.length())
+		{
+			s1 = lowercase(s1);
+			s2 = lowercase(s2);
+			
+			for(int i=0; i < sSize; i++)
+			{
+				if(s1.charAt(i) != s2.charAt(i))
+					return 0;
+			}
+			return 1;
+		}
+		else
+			return 0;
+	}
+	
+	String flipCase(String s)
+	{
+		int ch;
+		String temp = new String();
+		
+		for(int i=0; i< s.length(); i++)
+		{
+			ch = (int)s.charAt(i);
+			
+			
+			if(ch >= 65 && ch <= 90)
```

Comment:
Constants should be made for values like this.

Reply:
Done sir!